### PR TITLE
ASTT-108 Update readthedocs

### PR DESCRIPTION
**Pull request by AbednigoLethole**
The purpose of this work is to make the docs readable to users and document developer practices for the ASTT-CAM-software. Some Additions to the docs came from a recommendation. See the attached document: https://docs.google.com/document/d/16zpEhkW8wz56dh2Hhz27w0mbeQSh7vxXZiI_IbPiF5Q/edit

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -24,10 +26,13 @@ The system utilizes GPS data to determine real-time location information, such a
...
-   Software<software/index>
-   User Guide<how_to>
-   Comp Manager<component_manager_api>
+   Getting started<getting_started>
+   Installation<how_to_install>
+   User Guide<usage>
+   Developer Guide<dev_practices>
+   GUI API<gui/index>
+   Comp Manager API<component_manager_api>
+   Additional API<software/index>
```
Comment (USilere) [inline docs/index.rst:35]: This is amazing, just remove the starting the network part if we do not need it.

File: docs/index.rst [hunk 2]
```diff
@@ -24,10 +26,13 @@ The system utilizes GPS data to determine real-time location information, such a
...
 
-   GUI<gui/index>
-   Software<software/index>
-   User Guide<how_to>
-   Comp Manager<component_manager_api>
+   Getting started<getting_started>
+   Installation<how_to_install>
+   User Guide<usage>
+   Developer Guide<dev_practices>
+   GUI API<gui/index>
```
[the remark this answers is not in the document]
Comment (USilere) [reply docs/index.rst:33]: I assume how the dish enums info was fleshed out is how we would like to flesh out the Comp manager's info.

**Review by USilere — COMMENTED** [review]
Awesome stuff man, resolve the comments I left and I will approve. Thanks for the amazing work.

File: docs/index.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (AbednigoLethole) [reply docs/index.rst:33]: We will flesh out after the decoupling.

File: docs/index.rst (hunk 1 above)
Comment (AbednigoLethole) [reply docs/index.rst:35]: @USilere Please help me understand where you are seeing the network part. I am struggling to see it.

Comment (USilere) [reply docs/index.rst:35]: Additional API index

Comment (USilere) [reply docs/index.rst:35]: Please see screenshot

Comment (AbednigoLethole) [reply docs/index.rst:35]: I will remove it , Thanks.

**Comment by PhelokaziDube** [comment]
There's a part in GUI app where there's a module src.astt_gui.app module could you please flesh out what that module does instead.

**Review by USilere — APPROVED** [review]

**Pull request merged**